prody/database/pfam.py

```diff
@@ -473,36 +475,46 @@ def fetchPfamPDBs(**kwargs):
                 raise ValueError('Please provide an integer for start or end when using query.')
 
     from ftplib import FTP
-    data = []
+    data_file = open('pdbmap.gz','wb')
     ftp_host = 'ftp.ebi.ac.uk'
     ftp = FTP(ftp_host)
-    ftp.login('')
-    ftp.cwd('pub/databases/Pfam/mappings')
-    ftp.retrlines('RETR pdb_pfam_mapping.txt', data.append) 
+    ftp.login()
+    ftp.cwd('pub/databases/Pfam/current_release')
+    ftp.retrbinary('RETR pdbmap.gz', data_file.write)
+    ftp.quit()
+    data_file.close()
+
+    os.system('gunzip pdbmap.gz')
```

Comment:
This line is platform specific. It’s better to use python gunzip.

Reply:
I didn't find one but ok I'll have another look